async/src/main/java/org/odk/collect/async/TaskSpecWorker.kt

```diff
@@ -16,7 +16,7 @@ class TaskSpecWorker(
     override fun doWork(): Result {
         val cellularOnly = inputData.getBoolean(DATA_CELLULAR_ONLY, false)
         if (cellularOnly && connectivityProvider.currentNetwork != Scheduler.NetworkType.CELLULAR) {
-            Analytics.setUserProperty("EncounteredMeteredNonCellularInTasks", "true")
+            Analytics.setUserProperty("SawMeteredNonCellular", "true")
```

Comment:
I don't think we should make this change. It means different versions will report different values that we'll need to add up.

Reply:
Oh! Except we've never seen this property so I guess it's fine.